pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -292,6 +292,40 @@ class Build {
 
         return jdkRepo
     }
+
+    private String allocateEBCnodesForTest(testType) {
+        def test_target = buildConfig.TARGET_OS + '_' + buildConfig.ARCHITECTURE
+        // All targets are not supported via EBC yet. We need to test that target and add to default.json later on to make it enable
+        if ( ! DEFAULTS_JSON['testDetails']['ebcEnabledTargets'][test_target].find { it == testType } ) {
+            context.println "EBC does not support for $test_target->$testType yet!"
+            return ''
+        }
+
+        // context.library 'jenkins-ci-websphere' 
+        def group_label_UUID = 'Adopt-semeru_test_machine_' + UUID.randomUUID().toString()
```

Comment:
nit: Can remove `Adopt` and just call it `Semeru...`.

Reply:
changed to `semeru_aqatest_machine_`